lib/main_common.pm

```diff
@@ -135,6 +135,8 @@ sub init_main {
 
 sub loadtest {
     my ($test, %args) = @_;
+    my $ext = '.pm';
+    die "'$ext' is NOT allowed in path of loadtest!" if $test = $test . $ext;
```

Comment:
Please use `croak` instead. (`use Carp 'croak';`), so that we can know where the problem comes from, as the current message, comes from this line.

```
'.pm' is NOT allowed in path of loadtest! at /var/lib/openqa/cache/f40.suse.de/tests/sle/lib/main_common.pm line 138.
```

Reply:
updated, thanks